services/cinema_hall.py

```diff
@@ -0,0 +1,24 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
You should provide inner type of QuerySet. It makes your code more readable
```suggestion
def get_cinema_halls() -> QuerySet[CinemaHall]:
```